src/urbit/app/chess.hoon

```diff
@@ -317,7 +322,10 @@
               "no special draw available for game {<game-id.action>}"
               :~  [%give %poke-ack `~[leaf+err]]
               ==
-          :-  :~  :*  %give  %fact  ~[/game/(scot %da game-id.action)/moves]
+          :-  :~  :*  %give  %fact  ~[/archived-games]
+                      %chess-game-archived  !>(game.u.game-state)
+                  ==
+                  :*  %give  %fact  ~[/game/(scot %da game-id.action)/moves]
```

Comment:
This is functionally correct where it is, however I'd prefer it to be between the `%chess-game-result` and `%chess-update` cards.

I'm thinking of the ordering as:
1. Update opposing player
2. Update UI
3. Kick player and UI subscriptions